changelogs/current.yaml

```diff
@@ -1,7 +1,6 @@
-date: Pending
+date: July 18, 2024
```

Comment:
date is off by one

Reply:
yeah - i will recreate it again shortly - mostly i raise these to warm the cache/s